_includes/header.html

```diff
@@ -3,24 +3,22 @@
   <i class="icon icon-x-circle btn-mobile-close__icon hidden"></i>
 </span>
 
-<header class="panel-cover" style="background-image: url({{ site.baseurl }}/images/cover.jpg)">
+<header class="panel-cover">
   <div class="panel-main">
 
     <div class="panel-main__inner panel-inverted">
     <div class="panel-main__content">
         <a href="{{ site.baseurl }}/" title="link to home of {{ site.title }}">
-          <img src="{{ site.baseurl }}/images/profile.png" class="user-image" alt="My Profile Photo">
+          <img src="{{ site.baseurl }}/images/logo.png" class="crown-logo" alt="My Profile Photo">
```

Comment:
This doesn't exist so the image doesn't load

Reply:
Should be there now